flask_graphql/graphqlview.py

```diff
@@ -33,6 +33,7 @@ class GraphQLView(View):
     graphiql_template = None
     middleware = None
     batch = False
+    json_encoder = None
```

Comment:
Should this also include a [`json_decoder`](https://docs.python.org/2/library/json.html#json.JSONDecoder) member? Roughly equivalent change with the `json.loads` -> `self.json_decoder(…).decode(s)` a couple places below.